### DISCUSSION TITLE
What program do you use to create your diagrams?

### DISCUSSION THREAD

**Discussion by Orcomp** [thedataquarry/old-zola-site#13] [General]
The diagrams used in your blog posts are really nice. Just curious what program you use. Thanks

**Comment by prrao87** [comment]
I use [Excalidraw](https://excalidraw.com/). My mind was blown when I discovered it on other people's blogs a year ago, and I haven't looked back since 😀.. It's an open source project and I highly recommend following [their repo](https://github.com/excalidraw/excalidraw) and giving them a ⭐️ to show your support!
